homeassistant/components/webostv/quality_scale.yaml

```diff
@@ -0,0 +1,61 @@
+rules:
```

Comment:
Why do we throttle the state updates of the `async_update`?

Reply:
This is to protect against trying to connect to the TV to frequently when it is off. When the TV is off we try to connect and rely on timeout, if the user will force refresh when it is off it can lead to trying to connect before timeout expires